### PR TITLE
NTP and p2p fixes

### PR DESCRIPTION
**Pull request by iulianpascalau**
- made the ntp syncer ignore out of bound clock offset values
- made the p2p messenger less stringent about discarding too new messages

### CODE REVIEW

File: ntp/syncTime.go [hunk 1]
```diff
@@ -195,17 +197,19 @@ func (s *syncTime) sync() {
 
 	clockOffsetsWithoutEdges := s.getClockOffsetsWithoutEdges(clockOffsets)
 	clockOffsetHarmonicMean := s.getHarmonicMean(clockOffsetsWithoutEdges)
+	if clockOffsetHarmonicMean < -outOfBoundsDuration || clockOffsetHarmonicMean > outOfBoundsDuration {
```
Comment (sasurobert) [inline ntp/syncTime.go:200]: why don't you use abs ?

File: p2p/libp2p/netMessenger.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ const durationCheckConnections = time.Second
 const refreshPeersOnTopic = time.Second * 3
 const ttlPeersOnTopic = time.Second * 10
 const pubsubTimeCacheDuration = 10 * time.Minute
-const acceptMessagesInAdvanceDuration = 5 * time.Second //we are accepting the messages with timestamp in the future only fot this delta
+const acceptMessagesInAdvanceDuration = 20 * time.Second //we are accepting the messages with timestamp in the future only fot this delta
```
Comment (sasurobert) [inline p2p/libp2p/netMessenger.go:55]: only *for this delta - not fot

**Review by sasurobert — CHANGES_REQUESTED** [review]

**Review by SebastianMarian — APPROVED** [review]

File: ntp/syncTime.go (hunk 1 above)
Comment (iulianpascalau) [reply ntp/syncTime.go:200]: done, however, I have created a function for Abs because math.Abs operates on floats.

**Review by sasurobert — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed. 
Mention: Some of the cloud machines got stuck because of the low processing capabilities.

**Pull request merged**